cloudfront/outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "cloudfront_oai_iam_arn" {
   value = aws_cloudfront_origin_access_identity.origin_access_identity.iam_arn
 }
+
+output "cloudfront_domain_name" {
+  value = aws_cloudfront_distribution.cloudfront_s3_distribution.domain_name
+}
+
+output "cloudfront_hosted_zone_id" {
```

Comment:
Is it worth putting `s3` into the name of the output variable, just to make a little clearer?

Reply:
Same as for the comment above, this is the hosted zone id of the whole distribution.